### ISSUE TITLE
土壌硬度データの取り込みVBAをpythonで作成

### ISSUE THREAD

**Issue by yoshiyukiminami** [yoshiyukiminami/python#32]
土壌硬度計で収集したデータを所定のcsvファイルに取り込むVBAをpythonで作成する。
その際、硬度計の種類（DIK5531、DIK5532）を自動判別して取り込むようにする（現在は、それぞれ別のVBAで取り込み）

**Comment by duri0214** [comment]
@yoshiyukiminami 
基本的にはこの流れになるはずです
- python でCSVを読む
- CSVを変数上で整理する　`in_csv` から `out_csv` へ配列同士でガチャガチャ
- CSVで保存する

**Comment by yoshiyukiminami** [comment]
globでファイルパスを取得する際、「引数に recursive = True を指定することで、再帰的にファイルパス、フォルダパスを取得できます。」のような説明をしているサイトがありますが、再帰的をtrueにしておいて方が良いですか？

**Comment by duri0214** [comment]
@yoshiyukiminami  `recursive = true` でいいと思います。
https://note.nkmk.me/python-glob-usage/

再帰的な処理の大きな利点はなにかというと、例えば
ハウスが50個あってそれぞれ場所はいくらか離れていて、ハウスのなかに温度計があり、それぞれの室内温度を測ろうとしたときに、処理上「Xのハウスに入ったら室内温度を取得する」のような言い回しにできることです。（現実にはありえないけど、ハウスのなかにさらにハウスがあってもマトリョーシカのようなイメージで処理できるということです）

**Comment by duri0214** [comment]
そろそろ master にマージしないとひとつのブランチがあちこち修正しちゃっててすごいことになってそう

**Comment by yoshiyukiminami** [comment]
そうですね。issueはクローズしたのにmasterにmargeせずブランチのまま残っているのがほとんどですね。明日時間見つけて整理してみます。基本pullrequest出して岡田さんにmaster-margeしてもらうパターンで良かったですか？

**Comment by duri0214** [comment]
@yoshiyukiminami いったんオッケーって感じだったらそのまま自分でmaster-mergeしちゃっていいです。業務の場合は１つのｐｒに複数の修正を混ぜるな！とか言われますけど個人のソースであれば十分セルフ管理できるので
